WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -207,6 +212,51 @@ public async Task AccurateEstimationsAsync()
 			Assert.Equal(31, allFee.Estimations[1008]);
 		}
 
+		[Fact]
+		public async Task FixObviouslyWrongEstimationsAsync()
+		{
+			var rpc = CreateAndConfigureRpcClient(
+				estimator: target => target switch
+				{
+					2 => new FeeRate(70m),
+					3 => new FeeRate(10m),
+					6 => new FeeRate(3m),
+					18 => new FeeRate(1m),
+					36 => new FeeRate(1m),
+					1008 => new FeeRate(1m),
+					_ => throw new NoEstimationException(target)
+				},
+				hasPeersInfo: true
+			);
+
+			var allFee = await rpc.EstimateAllFeeAsync(EstimateSmartFeeMode.Conservative);
+			Assert.Equal(70, allFee.Estimations[2]);
+			Assert.True(allFee.Estimations[3] > 10);
+			Assert.True(allFee.Estimations[36] > 1);
+		}
+
+
+		[Theory]
+		[Repeat(1_000)]
+		public async Task ExhaustiveEstimationsAsync(int dummy)
+		{
+			var rpc = CreateAndConfigureRpcClient(
+				estimator: MemPoolInfoGenerator.GenerateFeeRateForTarget,
+				memPoolMinFee: 0.00037,
+				hasPeersInfo: true
+			);
+			rpc.OnGetMempoolInfoAsync = async () => await Task.FromResult(MemPoolInfoGenerator.GenerateMemPoolInfo());
+			var feeRates = await rpc.EstimateAllFeeAsync(EstimateSmartFeeMode.Conservative);
+			var estimations = feeRates.Estimations;
+
+			Assert.Equal(estimations.Count(), estimations.Distinct().Count());
```

Comment:
```suggestion
			Assert.Equal(estimations.Count, estimations.Distinct().Count());
```

This fixes a CA1829 warning.

Reply:
Done.